pages/en/using-mina/send-payment.mdx

```diff
@@ -101,7 +101,7 @@ Tip: You can run `mina client status` to see the current block height updating.
 
 ## Make a payment
 
-Finally we get to the good stuff, sending our first transaction! Before you send a payment, you'll need to unlock your account:
+Finally, we get to the good stuff–sending our first transaction! Before you send a payment, you'll need to unlock your account:
```

Comment:
This hyphen reads as a compound noun 'stuff-sending'. Can you replace it with a colon, or add spaces?
Typographically, I think this should also be an em-dash; using that symbol directly, or the common alternative of `--` would be better.

Reply:
It's already an em dash. But we can use the HTML entity to see if it shows differently on the website. `&mdash;`

Reply:
Actually, it looks ok on the preview: https://deploy-preview-65--mina-docs.netlify.app/en/using-mina/send-payment  But LMK if you prefer `&mdash;`

Reply:
So it is, sorry. I was fooled by the monospaced font 🤦‍♂️

Reply:
Yeah, I was baffled in VS Code for a moment before realizing the same earlier lol